src/client/components/pages/entities/edition-group.js

```diff
@@ -105,12 +125,20 @@ function EditionGroupDisplayPage({entity, identifierTypes, user}) {
 			{!entity.deleted &&
 			<React.Fragment>
 				<EditionTable editions={entity.editions} entity={entity}/>
-				<EntityLinks
-					entity={entity}
-					identifierTypes={identifierTypes}
-					urlPrefix={urlPrefix}
-				/>
-				<EntityRelatedCollections collections={entity.collections}/>
+				<Row>
+					<Col lg={8}>
+						<EntityLinks
+							entity={entity}
+							identifierTypes={identifierTypes}
+							urlPrefix={urlPrefix}
+						/>
+						<EntityRelatedCollections collections={entity.collections}/>
+
+					</Col>
+					<Col lg={4}>
+						<EntityReviews entity={entity} />
```

Comment:
We can directly pass the props we need rather than the whole entity object to avoid unnecessary rerenders.
Not sure it will have a big impact here but it's best practice:
```suggestion
						<EntityReviews entity={entity.reviews} entityType={entity.type} />
```

Or even, after refactoring the code as I suggest in another comment:
```suggestion
						<EntityReviews entity={entity.reviews} cbEntityType={getReviewEntityNameFromType (entity.type)} />
```